### PR TITLE
Feat/api calls

### PR DESCRIPTION
**Pull request by Aditi0505**
# Change Logs:

## Description:

- Added API calls for Cart and Wishlist functionality

## Screenshot 
- UI wise no change

## Files to be ignored while reviewing

-  netlify.toml

## Preview Link:

https://deploy-preview-8--redcloset.netlify.app/

### CODE REVIEW

File: src/pages/Cart/Cart.jsx [hunk 1]
```diff
@@ -2,17 +2,35 @@ import { useEffect } from "react";
 import { CartProduct } from "../../components";
 import { useCart, useToast } from "../../context";
 import { setTitle } from "../../utils/set-title";
```
Comment (rohanmathur91) [inline src/pages/Cart/Cart.jsx:4]: try barrel exports

**Review by rohanmathur91 — APPROVED** [review]
- Clean and readable code
- Having services in separate files is a good idea
- Overall looks good

File: src/pages/Cart/Cart.jsx (hunk 1 above)
Comment (Aditi0505) [reply src/pages/Cart/Cart.jsx:4]: Sure, I will add that.

**Review by hiteshverma27 — APPROVED** [review]
- Having a separate folder and files for services make it more manageable
- Good folder structure 
- Clean and readable code

**Pull request merged**